statler_api/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.conf.urls import url
-
-# Reference the views contained in this app. Will the API actually use views?
+# Reference the views contained in this app.
 from . import views
 
 # This is where the magical regex mapping happens.
 urlpatterns = [
-    url(r'^health-check$', views.healthCheck, name='healthCheck'),
-]
\ No newline at end of file
+    # urls that look like /api/play-list/ are forwarded to
+    # the views.getPlayList function
+    url(r'^play-list/$', views.getPlayList, name='play-list'),
```

Comment:
Sure, but the code is already so concise it's almost as short as constant definitions